autogpt/memory/llama_index.py

```diff
@@ -0,0 +1,112 @@
+"""LlamaIndex memory storage provider."""
+from llama_index.indices.vector_store.base import GPTVectorStoreIndex
```

Comment:
Does this require a package from pip?  If so should be in requirements.txt

Reply:
Just checking, I'd put the llama-index import in requirements.txt? 

Taking a look at `memory/__init__.py` as an example, it seems like imports for different memory modules are lazily loaded. E.g. for Milvus, pymilvus isn't in requirments.txt. 

Want to double-check if lazy import is ok or you'd want to make llama-index a required import

Reply:
Yeah we should not add this to requirements.